ouroboros-network-protocols/testlib/Ouroboros/Network/Protocol/ChainSync/Direct.hs

```diff
@@ -23,13 +23,13 @@ direct_ :: Monad m
         -> ClientStIdle header point tip m b
         -> m (a, b)
 direct_  ServerStIdle{recvMsgRequestNext}
-        (Client.SendMsgRequestNext stNext stAwait) = do
+        (Client.SendMsgRequestNext stAwait stNext) = do
     mresp <- recvMsgRequestNext
     case mresp of
       Left  resp    -> directStNext resp stNext
       Right waiting -> do resp <- waiting
-                          stNext' <- stAwait
-                          directStNext resp stNext'
+                          stAwait
```

Comment:
These two could also be the other way around.  Normally they are executed in parallel on different machines so it shouldn't matter.

Reply:
I'm going to leave it in the order it was---I don't think I have a preference.